### PR TITLE
marshalBadControllerEvent argument type hinting

### PR DESCRIPTION
**Pull request by autowp**
Exception -> Throwable

At line 102 marshalBadControllerEvent used with Throwable argument type

### CODE REVIEW

File: src/DispatchListener.php [hunk 1]
```diff
@@ -216,7 +216,7 @@ protected function marshalBadControllerEvent(
         $controllerName,
         MvcEvent $event,
         Application $application,
-        \Exception $exception
+        \Throwable $exception
```
Comment (michalbundyra) [inline src/DispatchListener.php:219]: it is not gonna work on php 5.6 :(
please have a look in [`composer.json`](https://github.com/zendframework/zend-mvc/blob/master/composer.json#L16), and it still must work on: `"php": "^5.6 || ^7.0"`

**Comment by michalbundyra** [comment]
it was already resolved in release [2.7.9](https://github.com/zendframework/zend-mvc/releases/tag/release-2.7.9):
https://github.com/zendframework/zend-mvc/pull/153/files

should be also ported to release v3

/cc @weierophinney

File: src/DispatchListener.php (hunk 1 above)
Comment (weierophinney) [inline src/DispatchListener.php:219]: @webimpress It works fine on PHP 5.6; the class does not need to exist for a typehint check.

Comment (weierophinney) [inline src/DispatchListener.php:219]: Missed that this was in the method signature and not the docblock. I'll remove the typehint during merge.

**Comment by weierophinney** [comment]
Thanks @autowp !

**Pull request merged**